spec/spec_helper.rb

```diff
@@ -60,6 +60,8 @@
   #     --seed 1234
   config.order = 'random'
 
+  Capybara.disable_animation = true
```

Comment:
Can you mention why we do this?

Reply:
do you think it is needed? It is a capybara option, which can be check directly in capybara. I am not sure we should duplicate information.

Reply:
well its not for what the commit message says. please either a comment or the right message.